### PR TITLE
If we have the capacity, try running Mono CI on ARM64

### PR DESCRIPTION
**Pull request by directhex**

### CODE REVIEW

File: eng/pipelines/runtime.yml [hunk 1]
```diff
@@ -570,7 +571,6 @@ jobs:
       testScope: innerloop
       liveRuntimeBuildConfig: release
       dependsOnTestBuildConfiguration: ${{ variables.debugOnPrReleaseOnRolling }}
-      dependsOnTestArchitecture: x64
```
Comment (safern) [inline eng/pipelines/runtime.yml]: You shouldn't remove this. The tests are not arch specific and are always built on x64.

**Comment by directhex** [comment]
ARM64 test run is GREEN :scream_cat:

**Review by jashook — APPROVED** [review]
We will need to watch the queue, but in general the change looks good

**Review by safern — COMMENTED** [review]
LGTM

**Pull request merged**